components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/stable/GitpodPortForwardingServiceImpl.kt

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package io.gitpod.jetbrains.remote.stable
+
+import com.jetbrains.rd.util.lifetime.Lifetime
+import com.jetbrains.rd.framework.util.launch
+import io.gitpod.jetbrains.remote.AbstractGitpodPortForwardingService
+import kotlinx.coroutines.CoroutineScope
+
+@Suppress("UnstableApiUsage")
+class GitpodPortForwardingServiceImpl : AbstractGitpodPortForwardingService() {
+    override fun runJob(lifetime: Lifetime, block: suspend CoroutineScope.() -> Unit) = lifetime.launch { block() }
```

Comment:
Why is it needed to make it abstract if in the end implementation is the same in stable and latest :thinking:

Reply:
They are not the same

```
import com.jetbrains.rd.framework.util.launch
```